sensors/weather/sensorhub-driver-meteobridge/build.gradle

```diff
@@ -16,6 +16,12 @@ osgi {
   manifest {
     attributes('Bundle-Vendor': 'Environment Systems')
     attributes('Bundle-Activator': 'uk.co.envsys.sensorhub.sensor.meteobridge.Activator')
+    attributes('Import-Package': '''
+      !java.*, !javax.crypto.*, !javax.imageio.*, !javax.net.*, !javax.naming.*, !javax.management.*, !javax.security.*, 
+      !javax.sql.*, !javax.swing.*, !javax.transaction.*, !javax.xml.*, !org.apache.avalon.*, !org.apache.log.*,
+      !org.apache.log4j.*, !org.apache.xerces.*, !org.w3c.dom.*, !org.xml.sax.*,!com.sun.*,
+      org.osgi.service.repository;resolution:=optional, 
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"

Reply:
Ok for building and installing: bff5b1598ceda162b07185739aaf0d11b18fc4c8